airflow/ui/src/components/ui/Status.tsx

```diff
@@ -28,12 +28,13 @@ import { stateColor } from "src/utils/stateColor";
 type StatusValue = DagRunState | TaskInstanceState;
 
 export type StatusProps = {
-  state?: StatusValue;
+  state: StatusValue | null;
 } & ChakraStatus.RootProps;
 
 export const Status = React.forwardRef<HTMLDivElement, StatusProps>(
   ({ children, state, ...rest }, ref) => {
-    const colorPalette = state === undefined ? "info" : stateColor[state];
+    // "null" is actually a string on stateColor
+    const colorPalette = stateColor[state ?? "null"];
```

Comment:
Shall we call this "created" perhaps? (I _might_ be making this change to the server side too, but it won't be universal just yet, so maybe we leave this as it is

Reply:
The entire `stateColor` object is a placeholder for now. So I'm happy to update it if we change it on the server side too.